README.md

```diff
@@ -47,6 +47,7 @@ It's modeled after Python's `json.tool`, reading from stdin and writing to stdou
   * [The GraphQL schema](#the-graphql-schema)
   * [Execution model](#execution-model)
   * [Miscellaneous](#miscellaneous)
+     * [Traversing within `@optional`](#traversing-within-optional-optional-)
```

Comment:
why the trailing `-` in the link?

Reply:
I believe it is because of the trailing backtick, but this is the correct link.

Reply:
More issues with my browser. Will fix.